instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/internal/TracingExecutionInterceptor.java

```diff
@@ -128,6 +131,7 @@ public TracingExecutionInterceptor(
   }
 
   @Override
+  @SuppressWarnings("deprecation") // need to access deprecated signer
```

Comment:
Even with `compileOnly`, the compile versions of the other artifacts also get bumped causing some new deprecations